cmd/geoipupdate/args.go

```diff
@@ -19,10 +19,15 @@ type Args struct {
 }
 
 func getArgs() *Args {
+	confFileDefault := vars.DefaultConfigFile
```

Comment:
This logic is here instead of `pkg/config` because we can't tell if the path passed to `config.go NewConfig(path, ...)` came from the command line (and thus we shouldn't override it)